client/src/components/Address/Address.js

```diff
@@ -15,6 +15,19 @@ import { useNavigate } from "react-router-dom";
 import "./address.css";
 import { getJWT } from "../../utils/jwt";
 import { parseJwt } from "../../utils/jwt";
+function loadRazorPay(src) {
+  return new Promise((resolve) => {
+    const script = document.createElement("script");
+    script.src = src;
+    script.onload = () => {
+      resolve(true);
+    };
+    script.onerror = () => {
+      resolve(false);
+    };
+    document.body.appendChild(script);
+  });
+}
```

Comment:
I was asking about this code. This is used to add a script from razorpay right?
Instead of loading the script can we use a npm and load it using import like we do in react

Reply:
@Gangotri-sg3 can you change this? so we can merge?

Reply:
I have checked the razor pay documentation,But the integration with web or react is done like this ,package is used only for node. If u know Please let me know, I will try to change it.